test/mpi/f77/pt2pt/pssendf.f

```diff
@@ -74,16 +75,16 @@ subroutine test_pair_pssend( comm, errs )
 C
          call MPI_Startall(2, requests, ierr)
 C
-         index = -1
-         do while (index .ne. 1)
+         completed = 0
+         do while (completed .lt. 2)
```

Comment:
IMO, same goes for here. Use waitall and remove the busy-loop.

Reply:
Actually scratch that. I think the intent of the test was to use these functions, so we should keep them like they are. I'm OK with the fix and will approve.

Reply:
Exactly. I myself actually rewrote these using `Waitall` at the first time, but then I thought that it would be better to keep use cases for `Waitany` etc, which are not many in the test cases and typical benchmarks.